com.github.aldrinzigmundv.MoonLight.json

```diff
@@ -0,0 +1,41 @@
+{
+    "app-id": "com.github.aldrinzigmundv.MoonLight",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "41",
+    "sdk": "org.gnome.Sdk",
+    "sdk-extensions": ["org.freedesktop.Sdk.Extension.rust-stable"],
+    "command": "moonlight",
+    "finish-args" : [
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--env=RUST_LOG=moonlight=debug",
+        "--env=G_MESSAGES_DEBUG=none",
+        "--env=RUST_BACKTRACE=1"
+    ],
+    "build-options" : {
+        "append-path" : "/usr/lib/sdk/rust-stable/bin",
+        "build-args" : [
+            "--share=network"
```

Comment:
this will fail there is no network.

Two options:
1. use flatpak-builder-tools to generate the cargo dependencies
2. use the tarball as the meson template will vendor the cargo crates. 

I recommend option 2 it available.